code/frameworks/angular/package.json

```diff
@@ -45,7 +45,6 @@
     "@storybook/csf": "0.0.2--canary.0899bb7.0",
     "@storybook/docs-tools": "7.0.0-alpha.34",
     "@storybook/node-logger": "7.0.0-alpha.34",
-    "@storybook/semver": "^7.3.2",
```

Comment:
add missing semver dependency

Reply:
you're right, I think it needs to be here.